lib/grinder.dart

```diff
@@ -73,6 +72,17 @@ Future startGrinder(List<String> args) {
   return handleArgs(args);
 }
 
+// Zone variables.
+
+/// Get the [GrinderContext] for the currently executing task.
+GrinderContext get context => zonedContext.value;
+
+/// Log an informational message to Grinder's output.
+void log(String message) => context.log(message);
```

Comment:
If we're exposing `log` and `fail` at the top-levl, we could expose `task` (possibly renamed to `currentTask` or `GrinderTask.current`) and `grinder` (or `Grinder.current`), and then deprecate GrinderTask altogether.  What do you think?